README.md

```diff
@@ -1,168 +1,88 @@
-
-### Create examples and tests
-
-Add one or more examples in the `examples` directory that consume your new module, and configure tests for them in the `tests` directory. For more information about tests, see [Tests](https://terraform-ibm-modules.github.io/documentation/#/tests).
-
-### Update the content in the readme file
-
-After you implement the logic for your module and create examples and tests, update this readme file in your repository by following these steps:
-
-1.  Update the title heading and add a description about your module.
-1.  Update the badge links.
-1.  Remove all the content in this H2 heading section.
-1.  Complete the [Usage](#usage) and [Required IAM access policies](#required-iam-access-policies) sections. The [Examples](#examples) and [Requirements](#requirements) section are populated by a pre-commit hook.
-
-### Commit your code and submit your module for review
-
-1.  Before you commit any code, review [Contributing to the IBM Cloud Terraform modules project](https://terraform-ibm-modules.github.io/documentation/#/contribute-module) in the project documentation.
-1.  Create a pull request for review.
-
-### Post-merge steps
-
-After the first PR for your module is merged, follow these post-merge steps:
-
-1.  Create a PR to enable the upgrade test by removing the `t.Skip` line in `tests/pr_test.go`.
-
-<!-- Remove the content in this previous H2 heading -->
-## Reference architectures
-
-<!--
-Add links to any reference architectures for this module.
-(Usually in the `/reference-architectures` directory.)
-See "Reference architecture" in Authoring Guidelines in the public documentation at
-https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
--->
+With this module you can create the following infrastructure:
+- Add security rules to a new or existing security group
+- Optionally create a new security group
+- Optionally add a list of security rules corresponding to the internal IBM flows
+- Optionally attach a security group to one or several targets (network interface, VPC load balancer, virtual private endpoint gateways, VPC VPN servers)
 
 ## Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
 ```hcl
-
+module "create_sgr_rule" {
+  source               = "git::https://github.com/terraform-ibm-modules/terraform-ibm-security-group"
+  security_group_rules = [{
+    name      = "allow-all-inbound"
+    direction = "inbound"
+    remote    = "0.0.0.0/0"
+  }]
+  target_ids            = ["r006-37e5b107-3006-480b-a340-bb1951357a73"]
```

Comment:
Did you double check that this is the right id format? and that it is not expecting a crn here?

Reply:
Yes, this is the right format. The regex validation` (^[-0-9a-z_]+$)` on the `target` from the provider `ibm_is_security_group_target` confirms CRN is not expected here.